_locales/de/messages.json

```diff
@@ -189,6 +189,10 @@
     "description": "Label for checkbox to filter tabs that are playing audio",
     "message": "Tabs die Ton abspielen nicht schliessen"
   },
+  "options_option_filterGroupedTabs_label": {
+    "description": "Label for checkbox to lock tabs that are in a tab group",
+    "message": "Schließen Sie nicht Registerkarten, die in einer Gruppe sind,"
+  },
```

Comment:
For any language you translated with Google Translate you can leave them out of messages.json (happy to take translations for any languages you're fluent in though). There is a [Tab Wrangler project on Crowdin](https://crowdin.com/project/tab-wrangler) where translations are managed. Contributors have popped in there occasionally to do human translations.

Reply:
Ack. Removed!